apps/galaxy/src/pages/_app.tsx

```diff
@@ -8,7 +8,9 @@ export default function App({ Component, pageProps }: AppProps) {
 
   return (
     <ThemeProvider theme={randomTheme}>
-      <Component {...pageProps} />
+      <Layout>
```

Comment:
Looking at this again, I feel like `<GlobalLayout>` or `<AppLayout>` delivers a better intent of this component?